queue_services/entity-digital-credentials/requirements.txt

```diff
@@ -22,4 +22,6 @@ urllib3==1.26.11
 Werkzeug==1.0.1
 git+https://github.com/bcgov/business-schemas.git@2.18.15#egg=registry_schemas
 git+https://github.com/bcgov/lear.git#egg=entity_queue_common&subdirectory=queue_services/common
-git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
+# git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
+git+https://github.com/chenhongjing/lear.git@22827-sql-versioning#egg=legal_api&subdirectory=legal-api
+git+https://github.com/chenhongjing/lear.git@22827-sql-versioning#egg=sql-versioning&subdirectory=python/common/sql-versioning
```

Comment:
So when we are ready to merge, what do all the queue service requirements file look like?

Will it just be a reference to `git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api`?

Reply:
We need to add a new dependency - sql-versioning (reference: [failed CI](https://github.com/bcgov/lear/actions/runs/11283040290/job/31381712511)), so it will be
```
git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
git+https://github.com/bcgov/lear.git#egg=sql-versioning&subdirectory=python/common/sql-versioning
```